src/test/java/io/jenkins/plugins/opentelemetry/backend/elastic/ElasticStack.java

```
@@ -58,70 +46,65 @@ public class ElasticStack extends DockerComposeContainer<ElasticStack> {
-        return "http://" + this.getServiceHost(ELASTICSEARCH_SERVICE, ELASTICSEARCH_PORT) + ":"
-                + this.getServicePort(ELASTICSEARCH_SERVICE, ELASTICSEARCH_PORT);
+        return "http://localhost:" + this.getServicePort(ELASTICSEARCH_SERVICE, ELASTICSEARCH_PORT);
     }
 
     /**
      * @return The URL to access to the Kibana Docker container.
      */
     public String getKibanaUrl() {
-        return "http://" + this.getServiceHost(KIBANA_SERVICE, KIBANA_PORT) + ":"
-                + this.getServicePort(KIBANA_SERVICE, KIBANA_PORT);
+        return "http://localhost:" + this.getServicePort(KIBANA_SERVICE, KIBANA_PORT);
     }
 
     /**
      * @return The URL to access to the OpenTelemetry Docker container.
      */
     public String getFleetUrl() {
-        return "http://" + this.getServiceHost(EDOT_SERVICE, OTEL_PORT) + ":"
-                + this.getServicePort(EDOT_SERVICE, OTEL_PORT);
+        return "http://localhost:" + this.getServicePort(EDOT_SERVICE, OTEL_PORT);
     }
 
     /**
-     * Create the index {@link #INDEX} fot testing in Elasticsearch.
+     * Create the index {@link #INDEX} for testing in Elasticsearch.
      *
      * @throws IOException
      */
-    public void createLogIndex() throws IOException {
-        Rest5Client restClient = getBuilder().build();
-        Rest5ClientTransport elasticsearchTransport = new Rest5ClientTransport(restClient, new JacksonJsonpMapper());
-        try (ElasticsearchClient client = new ElasticsearchClient(elasticsearchTransport)) {
-            client.indices()
-                    .create(new CreateIndexRequest.Builder().index(INDEX).build());
+    public void createLogIndexIfNeeded() throws IOException {
+        try (ElasticsearchClient client =
+                ElasticsearchClient.of(b -> b.host(getEsUrl()).usernameAndPassword(USER_NAME, PASSWORD))) {
+            try {
+                client.ping();
+            } catch (Exception e) {
+                return;
```

Why hide the exception?